### ISSUE TITLE
Image borders failing when packaged

### ISSUE THREAD

**Issue by jasonrclark** [shoes/shoes4#1428]
```
~/source/shoes4:bin/shoes package --jar samples/simple_border_image.rb
Packaging jar...
rm -f pkg/simple_border_image.jar
Creating pkg/simple_border_image.jar

~/source/shoes4:java -XstartOnFirstThread -jar samples/pkg/simple_border_image.jar
Ignoring rainbow-2.2.1 because its extensions are not built.  Try: gem pristine rainbow --version 2.2.1
SWALLOWED PAINT EXCEPTION ON #<Shoes::Swt::Border:0x3dfd6220> - go take care of it: i/o error (java.io.FileNotFoundException: uri:classloader:/gems/shoes-core-4.0.0.pre9/static/shoes-icon.png (No such file or directory))
Unfortunately we have to swallow it because it causes odd failures :(
SWALLOWED PAINT EXCEPTION ON #<Shoes::Swt::Oval:0x71936a92> - go take care of it: i/o error (java.io.FileNotFoundException: uri:classloader:/gems/shoes-core-4.0.0.pre9/static/shoes-icon.png (No such file or directory))
Unfortunately we have to swallow it because it causes odd failures :(
```

Need to have image borders play the same tempfile dance that we did for images themselves in #1427 

UPDATE: Bet this has the same relative file addressing issues I handled in #1429 too...

**Comment by PragTob** [comment]
:'(

**Comment by jasonrclark** [comment]
Closed by #1429
